### PR TITLE
docs: optifine compatibility FAQ

### PR DESCRIPTION
**Pull request by Lemmmy**
Adds an FAQ entry noting OptiFine's incompatibility with CC: Tweaked, and provides quick links to replacement mods for the most commonly used OptiFine features, as a quick reference for people who might think they need OptiFine specifically for one reason or another.

The Fabric links are provided mainly as a courtesy, even though the incompatibilities (as far as I'm aware) are only with Forge.

### CODE REVIEW

File: faqs/optifine.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### Fabric/Quilt
+- **[Sodium](<https://modrinth.com/mod/sodium>): performance improvements**
+- **[Indium](<https://modrinth.com/mod/indium>): mod compatibility, required with Sodium**
+- [Sodium Extra](<https://modrinth.com/mod/sodium-extra>): more settings for Sodium
+- [Iris](<https://modrinth.com/mod/iris>): shaders
+- [Continuity](<https://modrinth.com/mod/continuity>): connected textures
+- [Animatica](<https://modrinth.com/mod/animatica>): advanced resource pack texture features
+- [CEM](<https://modrinth.com/mod/cem>): custom entity models
+- [LambDynamicLights](<https://modrinth.com/mod/lambdynamiclights>): dynamic lights
```
Comment (SquidDev) [inline faqs/optifine.md:25]: I think we should probably *just* link the shader mods here, and leave everything else to the OF alternatives page. I'm always wary of having FAQs be too long. Sodium/Iris should work on NeoForge nowadays, so probably worth preferring those over Embeddium.

Maybe something like:

> For performance improvements, we recommend Sodium (for Fabric and NeoForge) or Embeddium (Forge). For shaders, Iris (Fabric/NeoForge) or Occulus (Forge) are recommended.
> 
> For Optifine's other features, view the full list of alternatives here.

Probably can be made a little less clunky.

**Review by SquidDev — COMMENTED** [review]
I'm somewhat surprised that people are still hitting this. CC:T *should* yell at you if Optifine is installed — is this message just not showing up for people?

**Comment by Lemmmy** [comment]
I'll ask the user that ran into issues today what version they were using exactly—I only know right now that it was on 1.20.1.

**Review by SquidDev — APPROVED** [review]
Thanks!

**Pull request merged**